apps/amo/context_processors.py

```diff
@@ -95,6 +95,8 @@ def global_settings(request):
              'href': reverse('devhub.themes.submit')},
             {'text': _('Developer Hub'),
              'href': reverse('devhub.index')},
+            {'text': _('Obtain an API Key'),
```

Comment:
This should be 'Manage API Keys' because we'll have other features later on like resetting the key